src/backend/parser/Make.js

```diff
@@ -0,0 +1,26 @@
+var JisonLex = require('jison-lex');
+var Jison = require('jison');
+var fs = require('fs');
+var path = require('path');
+
+var dest = path.resolve(process.argv[2]); // first command-line argument
+
+try { // create directory if it does not exist
+    fs.mkdirSync(dest);
+} catch(e) {
+    if (e.code !== 'EEXIST' ) throw e;
+}
+
+console.log(process.argv[0] + ' ' + process.argv[1] + ' ' +process.argv[2]);
+var lexerFilename = path.join(dest, 'ansic.lex.js');
+var parserFilename = path.join(dest, 'ansic.js');
+
+var lexerGrammar = fs.readFileSync('src/backend/parser/ansic.jisonlex', 'utf-8');
```

Comment:
It's just "fs.readFileSync("C:\Users\przemek\a.txt")" doesn't look too good and doesn't comply with general clean code rules - string constants should pretty much always be moved to constants. But please rather think why this script is necessary at all?